test/multiple-inputs.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-as %s -o %t1.bc
+; RUN: llvm-spirv %t.bc %t1.bc -spirv-text
+; RUN: FileCheck < %t.spt %s
+; RUN: FileCheck < %t1.spt %s
+; RUN: llvm-spirv %t.bc %t1.bc
+; RUN: spirv-val %t.spv
+; RUN: spirv-val %t1.spv
+; RUN: llvm-spirv -r %t.spv %t1.spv
+; RUN: llvm-spirv -s %t.bc %t1.bc
+; RUN: llvm-spirv -to-text %t.spv %t1.spv
```

Comment:
Why do you need these steps? You don't check LLVM IR or re-generated SPIR-V files content

Reply:
I wanted to check that multi-input mode works for all translation modes. I'll add more checks.

Reply:
Probably you can find some simple test (we have empty kernel or something like this) and use it as-is:
```
; RUN: llvm-as %S/empty.ll -o %t.1.bc
; RUN: llvm-as %S/empty.ll -o %t.2.bc
; RUN: llvm-spirv %t.1.bc t.2.bc
; RUN: llvm-spirv %t.1.spv t.2.spv -to-text
; RUN: FileCheck %S/empty.ll < %t.1.spt
; RUN: FileCheck %S/empty.ll < %t.2.spt
; etc.
```

Reply:
Done, thanks!